src/Boids.hs

```diff
@@ -22,7 +22,7 @@ data World = World { worldBoids :: [Boid]
 data Boid = Boid { boidPosition :: Point
                  , boidHeading :: Float
                  , boidSteer :: Float
-                 } deriving Show
+                 } deriving (Show, Eq, Ord)
```

Comment:
`Eq` for `Boid`? Where's [the comment I asked for two months ago?](https://github.com/tsoding/boids/pull/21#discussion_r91361969) :P